assets/css/style.css

```diff
@@ -76,15 +76,34 @@ ol {
 li {
   padding: 5px;
   display: flex;
+  justify-content: space-between;
+  align-items: center;
   gap: 1rem;
   list-style-position: inside;
   white-space: normal;
 }
 
+.leaderboard-heading {
+  display: flex;
+  justify-content: space-between;
+}
+
 .leaderboard-item {
   border-bottom: 1px solid var(--teritary-color);
 }
 
+#gold-medal {
+  color: goldenrod;
```

Comment:
Hi Kenny, can the color keywords be converted to hexadecimal values please? It keeps it consistent with other color calls throughout the file and also the root variables defined.

Reply:
Named colours converted to hex codes in latest commit